src/build.cmd

```diff
@@ -1,7 +1,7 @@
 @echo off
 if /i "%cswinrt_echo%" == "on" @echo on
 
-set CsWinRTBuildNetSDKVersion=6.0.424
+set CsWinRTBuildNetSDKVersion=8.0.303
 set CsWinRTBuildNet8SDKVersion=8.0.303

```

Comment:
technically we are installing it twice now, but if that is later going to change to .NET 9 that might be fine.

Reply:
Yeah I'm going to merge the .NET 9 PR right after this, so I figured we could just leave it 🙂